log4j-core-test/src/test/java/org/apache/logging/log4j/core/util/JsonReaderTest.java

```diff
@@ -375,6 +376,11 @@ public void test_nesting() {
                                 }})));
     }
 
+    @Test
```

Comment:
I'd prefer here a `@ParametrizedTest` touching different corner cases. Further, the following cases are not tested:
1. multi-line comment blocks (should pass)
1. not closed comment blocks (should fail)
1. not opened comment blocks (should fail)

Reply:
Added in https://github.com/apache/logging-log4j2/commit/e03f1832ed0791e5c489a65983de9ee707145ccd